web/src/main/java/com/navercorp/pinpoint/web/websocket/WebSocketHandlerDecoratorFactory.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2018 NAVER Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.navercorp.pinpoint.web.websocket;
+
+import org.springframework.web.socket.WebSocketHandler;
+
+/**
+ * @author minwoo.jung
+ */
+public interface WebSocketHandlerDecoratorFactory {
```

Comment:
There's a **org.springframework.web.socket.handler.WebSocketHandlerDecoratorFactory** interface that we can use instead of creating a new one?

Reply:
Ok~ I agree with you.